src/SwiftUI/SwiftUI/View.cs

```diff
@@ -32,7 +32,7 @@ struct CustomViewData
 	///  declares its return type as a concrete subclass of <see cref="View"/>
 	///   (it may not declare its return type as the <see cref="View"/> base class itself).
 	/// </remarks>
-	public unsafe abstract class View : SwiftStruct
+	public unsafe class View : SwiftStruct
```

Comment:
I don't think we need this change any longer

Reply:
Agreed!